tests/regression/test_multivariate.py

```
@@ -0,0 +1,310 @@
+
+
+def test_verdict_is_strict_pydantic_model(base_corr: np.ndarray, detector: CovarianceShiftDetector) -> None:
+    rng = np.random.default_rng(15)
+    x = _sigmoid_multivariate(30, base_corr, rng)
+    y = _sigmoid_multivariate(30, base_corr, rng)
+
+    verdict = detector.evaluate(candidate=y, baseline=x, baseline_id=uuid4(), run_id=uuid4())
+
+    assert isinstance(verdict, CovarianceShiftVerdict)
+    assert verdict.explanation, "explanation must be non-empty on every verdict"
+    with pytest.raises(Exception):
```

Ruff (0.16.3)

[warning] Do not assert blind exception: `Exception`

(B017)